src/main.ts

```diff
@@ -496,9 +502,7 @@ export default class WhatsApp {
   private qrCodeListeners: (() => void)[] = [];
 
   constructor(
-    qrPath = "./qrcode.png",
-    restoreSession = false,
-    keysPath = "./keys.json"
+    parameters = {}
```

Comment:
```suggestion
    opts = {      
        keysPath: "./keys.json",
        qrPath: "./qrcode.png",
        clientName: "WhatsApp forwarder",
        clientShortName: "WhatsAppForwarder",
        restoreSession: false 
    }
```
Rewriting all the options later is unnecessary.